Tests/Unit/ClientConfigurationTests.m

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+@import Foundation;
+
+#import "PFTestCase.h"
+#import "ParseClientConfiguration.h"
+#import "ParseClientConfiguration_Private.h"
+
+@interface ClientConfigurationTests : PFTestCase
+@end
+
+@implementation ClientConfigurationTests
```

Comment:
It looks like we are missing coverage on properties that apply when extensions data sharing is in use. Also, nitpicking - but maybe change tests name to `ParseClientConfigurationTests`.